### PR TITLE
change input type

### LINKED ISSUE #43: [FE] change date input
now we receive 10:00 when we input 10:00. 
but we need to receive date as ISO.

### PR DESCRIPTION
**Pull request by nagayamajun**
now input type about date is string.
but i receive it as ISO Format string.

### CODE REVIEW

File: apps/web/app/features/reservation/functions/customizeDateFormat.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const customizeDateFormat = (dateString: string) => {
```
Comment (nagayamajun) [inline apps/web/app/features/reservation/functions/customizeDateFormat.ts:1]: this func is to customize date for display

File: apps/web/app/features/reservation/functions/customizeDateFormat.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  const date = new Date(dateString);
+  const year = date.getFullYear();
+  const month = date.getMonth() + 1;
```
Comment (nanikasi) [inline apps/web/app/features/reservation/functions/customizeDateFormat.ts:4]: Why +1?

Comment (nanikasi) [reply apps/web/app/features/reservation/functions/customizeDateFormat.ts:4]: I got it.
See: https://qiita.com/pensaki/items/738810e07df108c61475

Comment (nanikasi) [reply apps/web/app/features/reservation/functions/customizeDateFormat.ts:4]: I want to change Date to dayjs.

**Review by nanikasi — APPROVED** [review]
LGTM

**Pull request merged**